xbuild/src/download.rs

```diff
@@ -138,9 +142,14 @@ impl<'a> DownloadManager<'a> {
                 self.macos_sdk()?;
             }
             Platform::Android => {
-                self.rustup_target("aarch64-linux-android")?;
-                self.android_ndk()?;
-                self.android_jar()?;
+                if Platform::host()? != Platform::Android {
+                    self.rustup_target("aarch64-linux-android")
+                        .context("rustup_target")?;
+                } else {
+                    info!("Skipping `rustup` for Android target");
+                }
```

Comment:
`termux` doesn't have `rustup`, but I'm not sure if we should generally skip this step if `platform() == Platform::host()?`, same for the architecture.